ide/libs.git/src/org/netbeans/libs/git/jgit/commands/IgnoreUnignoreCommand.java

```diff
@@ -105,7 +101,7 @@ protected void run () {
     private void changeIgnoreStatus (File f) throws IOException {
         File parent = f;
         boolean isDirectory = f.isDirectory() && (! Files.isSymbolicLink(f.toPath()));
-        StringBuilder sb = new StringBuilder('/');
+        StringBuilder sb = new StringBuilder();
```

Comment:
this did set the capacity of the SB. The char was cast to int. The code did still work though since it didn't need that slash.